pkgs/os-specific/linux/minimal-bootstrap/default.nix

```diff
@@ -144,9 +144,14 @@ lib.makeScope
     mes = lib.recurseIntoAttrs (callPackage ./mes { });
     mes-libc = callPackage ./mes/libc.nix { };
 
+    musl11 = callPackage ./musl/1.1.nix {
+      bash = bash_2_05;
```

Comment:
Btw Have you considered using busybox?

Reply:
The only busybox-based bootstrap project I know of is [bootstrap-from-tcc](https://github.com/ZilchOS/bootstrap-from-tcc) and none that have experimented with compiling it against mes-libc. Once you're out of mes-libc it's pretty much just as easy to compile bash+sed+grep as busybox. Amusingly we /do/ have a busybox-like tool in the bootstrap already, `heirloom`, but it's only needed as a sed implementation in a few places where meslibc-sed fails and before we can build glibc-sed. Once we switch sed to musl it will not be needed anymore and likely will be removed. 

The real game-changer to look out for is `gash` and `gash-utils`, the set of core utilities Guix uses written in scheme/guile. Work is underway to simplify it for the mes interpreter as well as improvements to mes with necessary features.